rawfile/bd2fs.py

```diff
@@ -251,22 +251,18 @@ def ControllerExpandVolume(self, request, context):
     def CreateSnapshot(self, request: csi_pb2.CreateSnapshotRequest, context):
         fs = None
         loop_dev = None
-        try:
-            file = img_file(request.source_volume_id)
-            loop_dev = attach_loop(file)
-            fs = from_device(loop_dev)
-            if not fs:
-                raise UnknownFileSystemError(
-                    device=loop_dev, volume_id=request.source_volume_id
-                )
-
-            fs.create_snapshot(name=request.name)
-            creation_time_ns = time.time_ns()
-            snapshot_id = f"{request.source_volume_id}/{request.name}"
-        finally:
-            if fs:
-                fs.unmount(clear_mountpoint=True)
```

Comment:
Why we do not unmount anymore?

Reply:
there's nothing to unmount here, the fs which was being caught was from the volume or none if the volume is not staged in the node